docs/docs/manual/installing.md

```diff
@@ -93,16 +93,16 @@ import TabItem from '@theme/TabItem';
 
 <TabItem value="win">
 
-1. On Windows 10, click the Windows start menu button, type "env," and look at the search results. Click “Edit the system environment variables.” (If you are using an earlier version of Windows, use the “Search” or “Search programs and files” box in the start menu.)
+1. On Windows 10, click the Windows start menu button, type <span class="menuItems">env</span>, and look at the search results. Click <span class="menuItems">Edit the system environment variables</span>. (If you are using an earlier version of Windows, use the <span class="menuItems">Search</span> or <span class="menuItems">Search programs and files</span> box in the start menu.)
```

Comment:
values (typed or displayed) should not use the **menuItems** class, but instead just the default Docusaurus markdown for code syntax such as `<code>env</code>`.  Which will stylize and beautify it out-of-the-box with a "white on grey" scheme.  See https://v2.docusaurus.io/docs/markdown-features/

So typed values or displayed values can just use the markdown `code` syntax.  And menu items can use the span class **menuItems**.

Reply:
The reason for not using <code> was to distinguish these UI items using a blue background for these texts.

Reply:
Sure, will remove the span and add code instead for values that are typed or displayed.

Reply:
@sjathin Also note, everything with Prism can be customized...  https://v2.docusaurus.io/docs/theme-classic/#codeblock  But I'm not finding the docs on how to style the single inline backticks for inline code syntax, versus the triple backticks (i.e. ```) on that page.  Hmm... see what you can find out.

Reply:
@sjathin And great job so far!  You rock!

Reply:
@thadguidry After checking the documentation of Docusaurus and checking with the dev team on slack, <code> tag is generated for both single and triple backticks. There is no differentiation between the two.

Reply:
Do y'all mean the difference between an inline triple backtick and a line-breaked triple backtick? 

```test```

```
test
```


I have been a bit lazy so far about differentiating between the two. But they are styled differently.

Reply:
@allanaaa the inline can use 1 set of backticks (not triple backticks, as those are usually reserved for codeblocks, not inline code)  And yes, you can actually do triple backticks on a line and it formats nicely on some applications depending on the markdown parser used, like here on GitHub :-)